### PR TITLE
Disco API - add some missing info

### PR DESCRIPTION
**Pull request by thomas-fossati**
Also, add the `alg` field to the verification key as per https://github.com/veraison/services/pull/131

### CODE REVIEW

File: api/well-known/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 The relevant resource is created in response to a client `GET`. The queried information is then outputted as shown in the examples below.
 
 ## Well-known API: Provisioning service
 The information for the provisioning service has the following attributes:
 
 * The allowed provisioning media types;
-* The version of the provisioning service.
+* The version of the provisioning service;
+* The current operational state of the service;
+* The exposed API endpoints and the corresponding URLs (relative to the base URL).
```
Comment (thomas-fossati) [inline api/well-known/README.md:12]:
```suggestion
* The exposed API endpoints and the corresponding URLs (relative to the request's base URL).
```

File: api/well-known/README.md [hunk 2]
```diff
@@ -42,7 +44,10 @@ The information for the verification service has the following attributes:
 
 * The public key used to verify the Attestation Result;
 * The allowed media types for attestation evidence;
-* The version of the verification service.
+* The version of the verification service;
+* The current operational state of the service;
+* The exposed API endpoints and the corresponding URLs (relative to the base URL).
```
Comment (thomas-fossati) [inline api/well-known/README.md:49]:
```suggestion
* The exposed API endpoints and the corresponding URLs (relative to the request's base URL).
```

**Review by SabreenKaur — COMMENTED** [review]
Would it be possible to change `state` to `service-state` in `docs/api/well-known/schemas/components.yaml` and in the above README.md changes as well?

File: api/well-known/schemas/components.yaml [hunk 3]
```diff
@@ -8,10 +8,9 @@ components:
         - api-endpoints
       properties:
         public-key:
```
Comment (SabreenKaur) [inline api/well-known/schemas/components.yaml:10]:
```suggestion
        ear-verification-key:
```

**Comment by SabreenKaur** [comment]
Change to:

File: api/well-known/schemas/components.yaml (hunk 3 above)
Comment (thomas-fossati) [reply api/well-known/schemas/components.yaml]: Done in [86b1a02](https://github.com/veraison/docs/pull/34/commits/86b1a02e0c1b647515f3d49c6b582fe8afa0c771)

**Pull request merged**